### ISSUE TITLE
error: no member named 'setprecision' in namespace 'std'

### ISSUE THREAD

**Issue by NAThompson** [boostorg/multiprecision#449]
In file included from main.cpp:2:
../multiprecision/include/boost/multiprecision/gmp.hpp:3278:15: error: no member named 'setprecision' in namespace 'std'
   os << std::setprecision(val.precision());
         ~~~~~^
1 error generated.

Fix is `#include <iomanip>` in `gmp.hpp`.

Discovered in standalone.

**Comment by NAThompson** [comment]
Fixed in #450

**Comment by ckormanyos** [comment]
Handled in #448
